js/index.js

```diff
@@ -1,10 +1,10 @@
-//creating a folder
+//creating a foooter
```

Comment:
Feedback totally optional here: 

The main idea when naming variables is to make comments unnecessary. The name itself should explain what it does. For instance, names like footer, year, and date don't exactly need any  comments because they're clear and simple enough to know what it is doing.

You might still want to use comments for really complex tasks to break them down, but usually, straightforward variable names do the job without extra explanation. It's up to you though, if you like adding comments for clarity and feel more comfortable, go ahead!"